tests/aws/services/apigateway/test_apigateway_basic.py

```diff
@@ -1836,24 +1771,32 @@ def test_rest_api_multi_region(
         type="AWS_PROXY",
         integrationHttpMethod="POST",
         uri=uri_us,
+        credentials=role_arn,
     )
+    apigateway_client_us.create_deployment(restApiId=api_us_id, stageName=stage_name)
 
-    # test valid authorization using bearer token
-    endpoint = url_function(api_eu_id, stage_name="local", path="/demo")
-    result = requests.get(endpoint, headers={}, verify=False)
-    assert result.status_code == 200
-    endpoint = url_function(api_us_id, stage_name="local", path="/demo")
-    result = requests.get(endpoint, headers={}, verify=False)
-    assert result.status_code == 200
+    def _invoke_url(url):
+        invoke_response = requests.get(url)
+        assert invoke_response.status_code == 200
+
+    endpoint = api_invoke_url(
+        api_eu_id, stage=stage_name, path="/demo", region="eu-west-1", url_type=url_type
+    )
+    retry(_invoke_url, retries=20, sleep=2, url=endpoint)
+    endpoint = api_invoke_url(
+        api_us_id, stage=stage_name, path="/demo", region="us-west-1", url_type=url_type
+    )
+    retry(_invoke_url, retries=20, sleep=2, url=endpoint)
 
     delete_rest_api(apigateway_client_eu, restApiId=api_eu_id)
     delete_rest_api(apigateway_client_us, restApiId=api_us_id)
-    aws_client_factory(region_name="eu-west-1").lambda_.delete_function(FunctionName=lambda_name)
-    aws_client_factory(region_name="us-west-1").lambda_.delete_function(FunctionName=lambda_name)
+    lambda_eu_west_1_client.delete_function(FunctionName=lambda_name)
+    lambda_us_west_1_client.delete_function(FunctionName=lambda_name)
 
 
 class TestIntegrations:
-    @markers.aws.unknown
+    @markers.aws.needs_fixing
+    # Attempts to create invalid integration with proxy
```

Comment:
oh yes, this is a bad one 😭 we have some reports about the S3 integration, I had started a PR a long time ago and never finished it 😅